scripts/compare_benchmarks.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+# -*- coding: utf-8 -*-
```

Comment:
On a more serious note, I'm finding it hard to give a narrative for those numbers. Personally, I work with statements like "TPC-H x became 20% faster / twice as fast". `+6%` was clear - it is now 6% faster. What does "1.0587" mean? I end up translating those numbers back into something I can understand, and will get it wrong for red numbers. I wonder if this might be the case for most people.

For single queries, I have no problem understanding that the old "-50%" and "+100%" mean the same thing, just in opposite directions. The problem only comes when those are, quite erroneously, averaged. I wonder if using the factor only for the average might make things easier.

Furthermore, I wonder if we would break anything by subtracting 1. I expect most people to understand red as negative, so having a scale `[-infty, -1], [1, infty]` is confusing.

Reply:
Ah, I just understood that your suggestion is to keep both columns. Isn't that more confusing for people who have not seen this discussion? The table should answer the question if the PR improved things or not without a statistics degree.

Reply:
Personally, I understand 17.04 much better than 1804%. And yes, I don't want to replace the percentages, they are especially good to get a quick first impression. I still see a problem for the average reader that -80% look by far less significant than its positive pendent.

Don't get what you mean by substracting 1. Where exactly?

Reply:
> Personally, I understand 17.04 much better than 1804%.

Mmmh, but wouldn't it be 1604%?

With big numbers, this is easier, because it is x times faster. For me, it's small red numbers that I can't wrap my head around. Together with the green/red coloring and the arrows, I would expect people to relate this to a percentage change on first sight.

Maybe this is something that could also be taken to StackOverflow? The problem seems to be easy enough to define, and I can't believe that it has no general solution.